standalone-metastore/metastore-server/src/main/java/org/apache/hadoop/hive/metastore/leader/LeaderElectionContext.java

```diff
@@ -42,12 +42,12 @@ public class LeaderElectionContext {
    * For those tasks which belong to the same type, they will be running in the same leader.
    */
   public enum TTYPE {
-    HOUSEKEEPING(new TableName(Warehouse.DEFAULT_CATALOG_NAME, "sys",
-        "metastore_housekeeping_leader"), "housekeeping"),
-    WORKER(new TableName(Warehouse.DEFAULT_CATALOG_NAME, "sys",
-        "metastore_worker_leader"), "compactor_worker"),
-    ALWAYS_TASKS(new TableName(Warehouse.DEFAULT_CATALOG_NAME, "sys",
-        "metastore_always_tasks_leader"), "always_tasks");
+    HOUSEKEEPING(new TableName(Warehouse.DEFAULT_CATALOG_NAME, "__METASTORE_LEADER_ELECTION__",
```

Comment:
Shouldn't the dbName come from this config 'METASTORE_HOUSEKEEPING_LEADER_LOCK_NAMESPACE'?

Reply:
This is the default db unless `METASTORE_HOUSEKEEPING_LEADER_LOCK_NAMESPACE` configured to a non-empty string.